src/feature/featureUtil.ts

```diff
@@ -5,14 +5,19 @@ import { buildQueryString } from "../utils";
 import { FEATURE_PATHS } from "./featurePath";
 import { FeatureType, FeatureOptions } from "./featureType";
 
-const getFeatureURL = <F extends FeatureType>(
+const getFeatureURL = async <F extends FeatureType>(
   featureType: F,
   featureOptions: FeatureOptions<F>,
   fullscriptOptions: FullscriptOptions,
   frameId: string
-): string => {
+): Promise<string> => {
   const { publicKey, env, domain } = fullscriptOptions;
-  const queryString = buildQueryString({ ...featureOptions, publicKey, frameId });
+  const queryString = await buildQueryString({
+    ...featureOptions,
+    fullscriptOptions,
+    publicKey,
+    frameId,
+  });
```

Comment:
we're passing along `fullscriptOptions` to allow us make the tokenization API call to the proper endpoint